ocs_ci/ocs/resources/mcg.py

```diff
@@ -63,15 +64,59 @@ def s3_create_bucket(self, bucketname, region=config.ENV_DATA['region']):
             CreateBucketConfiguration={
                 'LocationConstraint': region
             }
-        )
+        ).name
+    s3_create_obc = s3_create_bucket
+
+    def oc_create_obc(self, bucketname):
```

Comment:
Since most of the methods in this class get bucketname, would it make sense to instantiate the class with a bucketname?

Reply:
I agree with the premise, but it's impossible to require it - since you can create multiple buckets with different names under the same MCG.
Maybe it'll be better to move all those functions to a third, `mcg_bucket` class?

Reply:
Having an mcg_bucket class makes sense. 
Let's consider this (doesn't have to be implemented in this PR)